### PR TITLE
Add `--inset` flag

### PR DESCRIPTION
**Pull request by ewwink**
Fixes [sindresorhus/capture-website#8](https://github.com/sindresorhus/capture-website/issues/8)

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -52,6 +52,7 @@ $ capture-website --help
     --authentication         Credentials for HTTP authentication
     --debug                  Show the browser window to see what it's doing
     --overwrite              Overwrite the destination file if it exists
+		--inset                  Ignore certain pixels at the top of the page, but if  \`--element\` is defined it will include pixel around it
```
Comment (sindresorhus) [inline readme.md]: Incorrect indentation

File: readme.md [hunk 2]
```diff
@@ -77,6 +78,8 @@ $ capture-website --help
     --user-agent="I love unicorns"
     --cookie="id=unicorn; Expires=Wed, 21 Oct 2018 07:28:00 GMT;"
     --authentication="username:password"
+		--inset=100
+	  --element=".main-content" --inset=-10
```
Comment (sindresorhus) [inline readme.md]: Incorrect indentation

File: test.js.md [hunk 3]
```diff
@@ -56,4 +56,5 @@ Generated by [AVA](https://ava.li).
       userAgent: 'I love unicorns',
       waitForElement: '#header',
       width: 1000,
+			inset: -10,
```
Comment (sindresorhus) [inline test.js.md]: Incorrect indentation

File: readme.md (hunk 2 above)
Comment (ewwink) [reply readme.md]: my bad, it was looking good in vs code, fixed.

**Comment by sindresorhus** [comment]
Closing as https://github.com/sindresorhus/capture-website/pull/14 was closed without explanation.

**Pull request closed without merging**